Index: TestGit.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>class TestGit{\n\tpublic static void main (String[] args){\n\t\tSystem.out.println(\"hello test 1!\");\n\n\t\tSystem.out.println(\"hello test t1!\");\n\t}\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/TestGit.java b/TestGit.java
--- a/TestGit.java	(revision 94313e5287941339563f90c92ca12d9e78189088)
+++ b/TestGit.java	(date 1640077343480)
@@ -3,5 +3,6 @@
 		System.out.println("hello test 1!");
 
 		System.out.println("hello test t1!");
+		//add sth by t1
 	}
 }
